sdk/python/tests/unit/permissions/auth/server/test_auth_registry_server.py

```diff
@@ -21,13 +20,6 @@ def start_registry_server(
 ):
     if "kubernetes" in auth_config:
         mock_utils._mock_kubernetes(request=request, monkeypatch=monkeypatch)
-    elif "oidc" in auth_config:
```

Comment:
are you sure we want to remove this option? it's easier to test than the IT requiring docker